astropy/units/quantity.py

```diff
@@ -1271,8 +1271,11 @@ def _wrap_function(self, function, *args, **kwargs):
             if unit is None:
                 unit = self.unit
 
-            if not (isinstance(out, Quantity) and
-                    out.__quantity_subclass__(unit)[0] is type(out)):
+            if(isinstance(out, Quantity) and
```

Comment:
But with a space one gets this annoyance that the continuation line lines up with the first statement in the clause. Looking again at PEP8, though, I see my solution is not one of the "allowed" ones. So, here, at least I just added a comment so that the statement is more clearly offset.